koji_wrapper/base.py

```diff
@@ -57,16 +68,14 @@ def archives(self, **kwargs):
         """
         :param **kwargs: Any valid named parameter accepted by the koji client's
             listArchives method:
-            https://pagure.io/koji/blob/master/f/hub/kojihub.py#_3950
+
+            https://pagure.io/koji/blob/master/f/hub/kojihub.py
         :returns: list of archives from koji
         """
         return self.session.listArchives(**kwargs)
 
     def build(self, nvr):
         """
-        This method wraps the koji client method getBuild:
```

Comment:
Did you intend to drop the comment of which koji client method this is wrapping?

Reply:
I did remove it on purpose, as we are not accepting arbitrary args that the underlying method takes, only nvr.  However, maybe it would make sense to leave it in for consistency and in case anyone wants to look at the koji docs